packages/shared/src/utils/helpers/format-response.ts

```diff
@@ -42,6 +42,40 @@ export const formatPortfolioPosition = (
     };
 };
 
+export const idv_error_statuses = Object.freeze({
+    poi_name_dob_mismatch: 'POI_NAME_DOB_MISMATCH',
+    poi_dob_mismatch: 'POI_DOB_MISMATCH',
+    poi_name_mismatch: 'POI_NAME_MISMATCH',
+    poi_expired: 'POI_EXPIRED',
+    poi_failed: 'POI_FAILED',
```

Comment:
can we move this to constant file?

Reply:
@amina-deriv,  what file are you referring to?
this is a file inside helpers, is it not suitable?

Reply:
yeah best to move it in a constant file and update the code to this 

export const idv_error_statuses = {
    poi_name_dob_mismatch: 'POI_NAME_DOB_MISMATCH',
    poi_dob_mismatch: 'POI_DOB_MISMATCH',
    poi_name_mismatch: 'POI_NAME_MISMATCH',
    poi_expired: 'POI_EXPIRED',
    poi_failed: 'POI_FAILED',
} as const;